bottlecap/src/config.rs

```diff
@@ -39,6 +96,7 @@ impl Default for Config {
             // General
             site: "datadoqhq.com".to_string(),
```

Comment:
maybe let's add a TODO to make sure we don't forget this?

Reply:
Yup will do

Reply:
oh this is Default sorry! Yes my bad, all good